RPerformanceTracking/Private/_RPTTracker.m

```diff
@@ -123,6 +124,28 @@ - (void)updateURL:(NSURL *)url trackingIdentifier:(uint_fast64_t)trackingIdentif
     }
 }
 
+- (void)sendResponseHeader:(NSDictionary *)responseHeader trackingIdentifier:(uint_fast64_t)trackingIdentifier
+{
+    NSString *sourceURL = responseHeader[@"link"];
```

Comment:
Why is `link` used for the name here? I think the loaded URL (see spec for name) should be the `NSURLResponse.URL`.

Either it could be an extra parameter in this method or you can pass in the response, instead of just the headers, and extract the URL here.

Reply:
`link` is the full-size image that the thumbnail links to so that's the wrong data to use as the `name` here.